quickrun.el

```diff
@@ -330,7 +330,7 @@
 
     ("go/go"  .  ((:command . "go")
                   (:exec    . ((lambda ()
-                                 (if (string-match-p "_test\\.go\\'" (buffer-name))
+                                 (if (string-match-p "_test\\.go\\'" (buffer-file-name))
```

Comment:
Maybe we should change to use `buffer-name` if `buffer-file-name` is invalid. Hence, will be like.

```el
(if (buffer-file-name) (buffer-file-name) (buffer-name))
```

What do you think?

Reply:
> Maybe we should change to use `buffer-name` if `buffer-file-name` is invalid. Hence, will be like.
> 
> ```emacs-lisp
> (if (buffer-file-name) (buffer-file-name) (buffer-name))
> ```
> 
> What do you think?

Good point. I amended the commit.